docs/DevGuide/ImportingData.md

```diff
@@ -0,0 +1,64 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Importing data {#dev_guide_importing}
+
+The `importers` namespace holds functionality for importing data into SpECTRE.
+We currently support loading volume data files in the same format that is
+written by the `observers`.
+
+## Importing volume data
+
+The `importers::VolumeDataReader` parallel component is responsible for loading
+volume data and distributing it to elements of one or multiple array parallel
+components. As a first step, make sure you have added the
+`importers::VolumeDataReader` to your `Metavariables::component_list`. Also make
+sure you have a `Metavariables::Phase` in which you will perform registration
+with the importer, and another for loading the data. Here's an example for such
+`Metavariables`:
+
+\snippet Test_VolumeDataReaderAlgorithm.hpp metavars
+
+To register elements of your array parallel component for receiving volume data,
+invoke the `importers::Actions::RegisterWithVolumeDataReader` action in the
+`phase_dependent_action_list`. Here's an example:
+
+\snippet Test_VolumeDataReaderAlgorithm.hpp register_action
+
+Now you're all set to load a data file. To do so, invoke the
+`importers::ThreadedActions::ReadVolumeData` action on the
+`importers::VolumeDataReader`:
+
+\snippet Test_VolumeDataReaderAlgorithm.hpp read_data_action
+
+\snippet Test_VolumeDataReaderAlgorithm.hpp invoke_readvoldata
+
+The example shows the `execute_next_phase` function of an array parallel
```

Comment:
How about something like:

In the first snippet the `read_element_data_action` type alias is set to a specific specialization of the generic `importers::ThreadedActions::ReadVolumeData`. The first template parameter to `ReadVolumeData` is the option tag (or is this just something to mark the group??) from the input file (see the first bullet below for more detail), the second template parameter is a typelist of the tags to import and fill with the volume data, the third template parameter is the action used to set/fill the data (usually `Actions::SetData`, see the second bullet below for more detail), and the last template parameter is the component on which `Actions::SetData` will be invoked to set the imported data. The `ReadVolumeData`/`read_element_data_action` should be invoked from the `execute_next_phase` function as in the second snippet just above.